### ISSUE TITLE
Support for Veloren models?

### ISSUE THREAD

**Issue by poVoq** [goxel2/goxel2#20]
See:

https://book.veloren.net/contributors/artists/voxel-models.html

Import of the example files kinda worked with the old Goxel, but not well enough to use it to contribute to Veloren.

**Comment by pegvin** [comment]
Can you describe "not well enough", like problems you're facing and stuff like that?

**Comment by pegvin** [comment]
hey there, i tried some models from the google drive listed on the website you've shared and it worked, can specify what is the problem you're facing?

**Comment by poVoq** [comment]
Indeed, it seems to work better now. However, I noticed when importing and exporting a weapon model that the point of origin changes.

**Comment by pegvin** [comment]
> Indeed, it seems to work better now. However, I noticed when importing and exporting a weapon model that the point of origin changes.

Okay, will take a look into it.
